docs/wowa/design-system-modal/mobile-brief.md

```
@@ -0,0 +1,223 @@
+      ),
+    ),
+  ],
+)
+```
+
+**핵심 변경점:**
+- `BoxDecoration` → `CustomPaint` + `SketchPainter`
+- `Border.all()` → `SketchPainter.showBorder` (불규칙 손그림 테두리)
+- 노이즈 텍스처 자동 활성화
+- 그림자: 별도 `SketchPainter` 레이어로 오프셋 그림자 구현
+
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)